packages/fx-core/src/plugins/resource/function/enums.ts

```diff
@@ -82,6 +82,7 @@ export enum FunctionEvent {
   skipDeploy = "skip-deploy",
   DeploymentInfoNotFound = "deployment-info-not-found",
   generateArmTemplates = "generate-arm-templates",
+  updateArmTemplates = "update-arm-templates",
```

Comment:
You add a new telemetry event here, do you help classify it in GDPR inventory?

Reply:
will add this telemetry event after all resource added